index.html

```diff
@@ -3,7 +3,7 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Document</title>
+    <title>Calculator</title>
   </head>
   <body></body>
```

Comment:
Right here, you should add an h1 element inside the body with maybe Hello World or something
You should also have the main element and give it role attribute and you should also link your js file too